src/bundle/Resources/views/themes/admin/content_type/field_definition.html.twig

```diff
@@ -28,8 +28,12 @@
         {% if field_definition.enabled is defined %}
             {{ form_row(field_definition.enabled) }}
         {% endif %}
-        {{ form_row(field_definition.name) }}
-        {{ form_row(field_definition.identifier) }}
+        {{ form_row(field_definition.name, {
+            attr: { 'data-autogenerate-identifier-target-selector': '.ibexa-ct-identifier-' ~ field_definition.vars.value.identifier }
+        }) }}
+        {{ form_row(field_definition.identifier, {
+            attr: { class: 'ibexa-ct-identifier-' ~ field_definition.vars.value.identifier  }
```

Comment:
```suggestion
            attr: { id: 'ibexa-ct-identifier-' ~ field_definition.vars.value.identifier  }
```
I'm not sure if we should set class, id might be better and would reflect more the purpose. WDYT?

Reply:
I agree that id is a better idea. I followed the usage example in the product type, so if we are going to change the class to id, I think it would be a good idea to change it in the product type forms as well.